src/network/networkd-link.c

```diff
@@ -3376,6 +3376,9 @@ int link_save(Link *link) {
                 char **dhcp_domains = NULL;
                 unsigned j;
 
+                fprintf(f, "REQUIRED_ONLINE=%s\n",
```

Comment:
Since the option is called "RequiredForOnline", please use the same internally, i.e. "REQUIRED_FOR_ONLINE=" here, and "link->network->required_for_online" below.